lib/engine/game/g_1846/meta.rb

```diff
@@ -40,6 +40,17 @@ module Meta
             players: [2, 3, 4, 5],
           },
         ].freeze
+
+        MUTEX_RULES = [
+          %i[first_ed second_ed_co],
+].freeze
+
+        def self.check_options(options, _min_players, _max_players)
```

Comment:
This method isn't needed any more, as the MUTEX_RULES mean it will be impossible to select both of these options. (Though of course it doesn't do any harm either).